master/docs/Makefile

```diff
@@ -10,8 +10,6 @@ TAR_TRANSFORM := $(if $(filter bsdtar,$(TAR_VERSION)),-s /^html/$(VERSION)/,--tr
 docs.tgz: clean html singlehtml images-png
 	sed -e 's!href="index.html#!href="#!g' < _build/singlehtml/index.html > _build/html/full.html
 	tar -C _build $(TAR_TRANSFORM) -zcf $@ html
-pipdeps:
```

Comment:
Removing pipdeps from the entire file - 
```
-html: conf.py pipdeps
+html:
```
I'm not sure of what should replace the ```pipdeps``` command. 
Frankly, I haven't been able to completely understand makefiles till now.